sorc/checkout.sh

```diff
@@ -29,9 +29,11 @@ mkdir -p ${logdir}
 
 echo ufs-weather-model checkout ...
 if [[ ! -d ufs_model.fd ]] ; then
-    git clone https://github.com/ufs-community/ufs-weather-model ufs_model.fd >> ${logdir}/checkout-ufs_model.log 2>&1
+    #git clone https://github.com/ufs-community/ufs-weather-model ufs_model.fd >> ${logdir}/checkout-ufs_model.log 2>&1
+    git clone https://github.com/DeniseWorthen/ufs-weather-model ufs_model.fd >> ${logdir}/checkout-ufs_model.log 2>&1
     cd ufs_model.fd
-    git checkout ${ufs_model_hash:-5c2d1a9}
+    #git checkout ${ufs_model_hash:-5c2d1a9}
+    git checkout feature/cmake_meshcapbuild
```

Comment:
This needs to be a hash or tag, not a branch

Reply:
Looks like you took care of this while I was reviewing.

Reply:
looks like the most recent code change changing this to a tag didn't get updated, it should be fixed with the merged in code now.